### DISCUSSION TITLE
Exclude directories using the configuration file

### DISCUSSION THREAD

**Discussion by denis-rossati** [composer-unused/composer-unused#421] [Ideas]
Today the user can include directories as he please in the configuration file through the CLI [as described in the docs](https://github.com/composer-unused/composer-unused#exclude-folders-and-packages)

It would be good if he could exclude directories and folders from being analyzed using the config file too:
```
<?php

declare(strict_types=1);

use ComposerUnused\ComposerUnused\Configuration\Configuration;
use Webmozart\Glob\Glob;

return static function (Configuration $config): Configuration {
    return $config->excludeDir(/* path */);
};
```

If isn't too much to ask, I would be more than happy to help with the PR.

**Comment by icanhazstring** [comment]
Hi. Thanks, that sounds like a great idea to have. This would actually also reflect the `--excludeDir` option on the command itself.

Which, however, right now doesn't seem to work  correctly (#418). Nevertheless, I would appreciate a PR  as I am currently pretty busy with work.

This would cover composer-unused and symbol-parser. As this path needs to be dragged along.
